modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -350,17 +440,17 @@ GEF<-function(setting,Forecast,Observed, H, extraArg, nitr=50000, nburnin=10000,
     
   }
 
-  dat <- runMCMC(Cmcmc, niter = nitr, nburnin=nburnin)
-  
+  dat <- runMCMC(Cmcmc, niter = 50000, nburnin=10000)
+  dat_save <- dat[(nrow(dat)-250):nrow(dat),]
+  save(dat_save, file = file.path(outdir, paste0('dat',t,'.Rdata')))
```

Comment:
I know it'd be too picky, but that'd be nice if you could save `dat` in SDA folder in `outdir` which we keep plots and other sda related outputs.